File: Js/fizzBuzz.js
for (let i = 0 ; i<=100;i++){
    let output = '';
    
    if(i%3 == 0) output +='Fizz';
    if(i % 5 == 0) output += 'Buzz';
    if (i % 7 == 0) output += 'Bazz';

    console.log(output || i+ ' no es multiplo de 3 ni 5 ni 7');
}

File: Python/fizz_buzz.py
#algoritmo que imprime fizz si un numero es multiplo de 3 y buzz si un numero es multilpo de 5

def fizz_buzz(n):
    for i in range(1, n+1):
        salida = ""
        if i%3 == 0:
            salida += "fizz"
        if i%5 == 0:
            salida += "buzz"
        
        print(salida or f"el numero {i} no es multiplo de  3 o 5")

fizz_buzz(100)